.github/workflows/release-published.yml

```diff
@@ -26,4 +26,29 @@ jobs:
           env:
             MAVEN_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
             MAVEN_PASSWORD: ${{ secrets.SONATYPE_TOKEN }}
-            GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
\ No newline at end of file
+            GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+    deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      outputs:
+        tag: ${{ steps.collect-data.outputs.tag }}
+        version: ${{ needs.setup.outputs.version }}
+      steps:
+        - name: Download liquibase mongodb xsd
+          uses: actions/checkout@v3
+          with:
+            # Relative path under $GITHUB_WORKSPACE to place the repository
+            path: liquibase-mongodb
+            repository: "liquibase/liquibase-mongodb"
+
+        - name: Upload to s3
+          env:
+            AWS_ACCESS_KEY_ID: ${{ secrets.LIQUIBASEORIGIN_ACCESS_KEY_ID }}
+            AWS_SECRET_ACCESS_KEY: ${{ secrets.LIQUIBASEORIGIN_SECRET_ACCESS_KEY }}
+            AWS_DEFAULT_REGION: us-east-1
+          # aws s3 sync syncs directories and S3 prefixes.
+          run: |
+            aws s3 sync liquibase-mongodb/src/main/resources/liquibase.parser.core.xml/ s3://liquibaseorg-origin/xml/ns/mongodb/ --content-type application/octet-stream --only-show-errors
```

Comment:
@KushnirykOleh 
Do we need to publish the initial scheme?

Reply:
i don't think we need it, it will create more confusion than help. I would just publish new files and let dbchangelog-ext.xsd stay in the jar